hw/ip/usb_fs_nb_pe/usbfs_nb_pe.core

```diff
@@ -2,7 +2,7 @@ CAPI=2:
 # Copyright lowRISC contributors.
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
-name: "lowrisc:ip:usbfs_nb_pe:0.1"
+name: "lowrisc:ip:usb_fs_nb_pe:0.1"
```

Comment:
This module is used in multiple places (usbdev.core and usbuart.core), you also need to adjust the name there.

Reply:
Jep, I've already done that (although those two changes are probably drowned by all the other diffs in this PR ;)).

I will split this preparatory renaming out into a different PR.

Reply:
Moved to #1312